task_reactredux/task1/src/containers/MenuContainer.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import Menu from '../views/Menu/index';
+
+class MenuContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: false,
+      showMenu: true
+    }
+  }
+
+  handleChange = (event, value) => {
+    this.setState({ value });
+  };
+  
+  componentDidMount() {
```

Comment:
Probably it's better to move this code into constructor and use code below for initializing state?